templates/configmaps_chatops.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-chatops-configs
+  annotations: 
+    description: Custom StackStorm chatops configs, shipped in '/opt/stackstorm/chatops/'
+  labels:
+    app: st2chatops
+    tier: backend
+    vendor: stackstorm
+    support: {{ template "supportMethod" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+# see configmaps_st2-conf.yaml for indepth explanantion of mappings here. 
```

Comment:
I figure the implementation is identical to st2.conf without having to apply layers. though I think it is. a good idea to have the first st.conf applies through the system and then users come on topic. As per described previously.